pyproject.toml

```diff
@@ -4,7 +4,8 @@ exclude = [
   "tools/cpplint.py",
   "tools/gyp",
   "tools/inspector_protocol",
-  "tools/eslint/node_modules",
+  "tools/eslint",
```

Comment:
This excludes more.  Why?

Reply:
> This excludes more.  Why?

It bugged me a bit that most of the excluded were `tools/dir` while this one was `tools/dir/subdir`. I can undo it if you'd like